src/main/javascript/view/ParameterView.js

```diff
@@ -71,23 +71,23 @@ SwaggerUi.Views.ParameterView = Backbone.View.extend({
   },
 
   // Return an appropriate template based on if the parameter is a list, readonly, required
-  template: function(){
-    if (this.model.isList) {
-      return Handlebars.templates.param_list;
-    } else {
-      if (this.options.readOnly) {
-        if (this.model.required) {
-          return Handlebars.templates.param_readonly_required;
-        } else {
-          return Handlebars.templates.param_readonly;
-        }
-      } else {
-        if (this.model.required) {
-          return Handlebars.templates.param_required;
-        } else {
+  template: function() {
+    // if (this.model.isList) {
```

Comment:
There used to be separate handlebar template files for each combination of readonly/editable and required/optional parameters, and this block of code selected which file to use. In this PR I deleted all those templates and made one that covers the different use cases using 🌈IF-STATEMENTS🌈. I left this block of code in the source file but just commented it out instead. 

Actually now that you mention it, I think it would be better to just delete the commented out parts now. I assume that's what you meant to ask? I'll push a fixup.